### PR TITLE
Update publication list

### PR DESCRIPTION
**Pull request by jshinm**

### CODE REVIEW

**Review by PSSF23 — CHANGES_REQUESTED** [review]
- All new entries lack URLs & author highlights (see existing as examples).
- Inconsistent journal labeling of `arXiv` articles (also see existing as examples). 
- "Modern Machine Learning: Partition & Vote" already exists.
- If "Eliminating accidental deviations to minimize generalization error with applications in connectomics and genomics" is peer-reviewed now, what is its new journal/publisher/URL?

**Comment by jshinm** [comment]
@PSSF23 Thank you so much for your thorough review! I've updated the bib file to address your concern

File: content/bibs/pubs.bib [hunk 1]
```diff
@@ -1338,15 +1338,49 @@ @article{Zheng2016b
...
+  url={https://www.nature.com/articles/s41597-021-00849-3},
+  volume={8},
+  number={1},
+  pages={1--9},
+  year={2021},
+  publisher={Nature Publishing Group},
+  keywords = {peer-reviewed}
+}
 
+@article{powell2021ten,
```
Comment (PSSF23) [inline content/bibs/pubs.bib]: I think you missed this one~

File: content/bibs/pubs.bib [hunk 2]
```diff
@@ -1522,19 +1556,34 @@ @article {Rose2021
...
+  pages={e1009279},
+  year={2021},
+  url = {https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1009279},
+  publisher={Public Library of Science San Francisco, CA USA},
+  keywords = {peer-reviewed}
+}
+
 @article {Bridgeford802629,
 	author = {Bridgeford, Eric W. and Wang, Shangsi and Yang, Zhi and Wang, Zeyi and Xu, Ting and Craddock, Cameron and Dey, Jayanta and Kiar, Gregory and Gray-Roncal, William and Coulantoni, Carlo and Douville, Christopher and Priebe, Carey E. and Caffo, Brian and Milham, Michael and Zuo, Xi-Nian and Vogelstein, Joshua T.},
-	author+an = {1=trainee; 16=highlight},
```
Comment (PSSF23) [inline content/bibs/pubs.bib:1527]: Why delete information here? I saw there's a peer-reviewed version of the article, so this entry could be completely replaced?

**Review by PSSF23 — COMMENTED** [review]
Looks great! Just have two minor issues.

File: content/bibs/pubs.bib (hunk 1 above)
Comment (jshinm) [reply content/bibs/pubs.bib]: Turns out there have been not just duplicates but triplicates perhaps even quad that could be found either by title and their codes. This particular one is the duplicate by code, and I deleted this one

File: content/bibs/pubs.bib (hunk 2 above)
Comment (jshinm) [reply content/bibs/pubs.bib:1527]: This one is the remnant of the duplicate from which I copied some information from. This is now deleted! Thank you so much for all these correction, @PSSF23

File: content/bibs/pubs.bib [hunk 3]
```diff
@@ -2197,22 +2222,31 @@ @inproceedings{Vogelstein2010b_2
...
+    title = {CloudReg: automatic terabyte-scale cross-modal brain volume registration},
+    journal = {Nature Methods},
+    year = {2021},
+    month = {7},
+    issn = {1548-7105},
+    doi = {10.1038/s41592-021-01218-z},
+    url = {https://doi.org/10.1038/s41592-021-01218-z},
+    eprint = {https://www.nature.com/articles/s41592-021-01218-z.pdf},
     keywords = {tech}
 }
```
Comment (PSSF23) [inline content/bibs/pubs.bib:2251]: `CloudReg` already exists as `peer-reviewed`, so this additional entry could be deleted?

**Review by PSSF23 — APPROVED** [review]
@jshinm Except the `CloudReg` thing I don't see any other problems for now. But as you noticed, there might still be duplicates out there. I'm afraid that those could only be solved by the "find one remove one" method...

File: content/bibs/pubs.bib (hunk 3 above)
Comment (jshinm) [reply content/bibs/pubs.bib:2251]: It seems like this tech report is peer-reviewed, so I generated two entries on purpose :)

**Comment by jshinm** [comment]
@PSSF23 I totally I agree. To your concern, I've created an [issue](https://github.com/neurodata/neurodata.io/issues/573) to ensure that we don't have any duplicates. I will make sure.

File: content/bibs/pubs.bib (hunk 3 above)
Comment (PSSF23) [reply content/bibs/pubs.bib:2251]: Would be better if keywords could be multiple items?

**Pull request merged**